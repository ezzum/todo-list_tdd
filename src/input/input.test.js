import { mount } from 'enzyme';
import Input from './input';

describe('Input tests', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(<Input />);
  });
  afterEach(() => {
    wrapper.unmount();
  });

  test('snapshot test', () => {
    expect(wrapper).toMatchSnapshot();
  });
  test('render input', () => {
    expect(wrapper.find('input')).toHaveLength(1);
  });
});
