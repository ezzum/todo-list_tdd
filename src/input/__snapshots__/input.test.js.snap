// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input tests snapshot test 1`] = `ReactWrapper {}`;
